zellij-server/src/wasm_vm.rs

```diff
@@ -219,6 +220,13 @@ fn host_open_file(plugin_env: &PluginEnv) {
         .unwrap();
 }
 
+fn host_switch_tab_to(plugin_env: &PluginEnv, tab_idx: u32) {
```

Comment:
Good for now! Hopefully we can add a way for plugins to dispatch actions directly soon!